common/transport/amqp/.npmignore

```diff
@@ -3,4 +3,5 @@ devdoc/
 test/
 samples/
 src/
-lib/**/*.map
\ No newline at end of file
+dist/**/*.map
+.nyc_output
```

Comment:
is this supposed to be added?

Reply:
It certainly seemed like the right think to do.  Consistency between all the directories.